charts/kkr/wallet-api/templates/ingress/ingress.api.public.yaml

```
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "wallet-api.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "wallet-api.labels" . | nindent 4 }}
+  annotations:
+    konghq.com/plugins: {{ $fullName }}-cors
```

```suggestion
    konghq.com/plugins: {{ include "wallet-api.cors" . }}
```
`_helpers.tpl`에서 선언한 값을 사용하는 것이 더 좋을 것 같습니다.

---

기시감이 들어서 다시 확인해보니, #2 에 리뷰한 내용이네요!